README.md

```diff
@@ -1,19 +1,10 @@
 # GRANNY Experiments
 
-This repo contains the experiments for the [GRANNY paper](
```

Comment:
Could you undo your changes in this file. Let's use the current upstream `README.md`.